docs/make.jl

```diff
@@ -24,6 +24,7 @@ function main()
             analytics = "UA-154489943-2",
         ),
         doctest = true,
+        strict = true,
```

Comment:
I think it might be because you haven't included those docstrings with at-docs/autodocs blocks. You could add `modules`:

```suggestion
        strict = true,
        modules = [CUDA],
```

and re-run, and see if you get the respective "missing docstring" errors. Although, this may also lead to a ton of false-positives, in case you have some internal docstrings that are not included in the manual.